modules/drivers/audio/speaker_component.h

```diff
@@ -0,0 +1,67 @@
+/******************************************************************************
+ * Copyright 2017 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+#pragma once
+
+#include <atomic>
+#include <future>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "modules/drivers/audio/proto/speaker_config.pb.h"
+#include "modules/drivers/audio/proto/audio.pb.h"
+#include "modules/drivers/audio/respeaker.h"
+#include "cyber/cyber.h"
```

Comment:
include order: `cyber/xxx.h` before `modules/xxx/xxx.h`.  Run `scripts/clang_format.sh` to do that.

Reply:
Thank you! Previously I just sorted them using my eyes... but this one is missed.
I have used `scripts/clang_format.sh` to format the four files according to your suggestion.